Gemfile

```diff
@@ -7,6 +7,11 @@ gemspec
 gem 'bump', require: false
 gem 'pry'
 gem 'rake', '~> 12.0'
+# Workaround for crack 0.4.3 or lower.
+# Depends on `rexml` until the release that includes
+# the following changes:
+# https://github.com/jnunemaker/crack/pull/62
+gem 'rexml'
```

Comment:
📝 I think that adding to Gemfile, not gemspec is eligible because crack is a gem on which WebMock depends.